lemur/users/service.py

```diff
@@ -14,6 +14,9 @@
 from lemur.users.models import User
 
 
+DEFAULT_ROLES = ["admin", "operator", "read-only"]
```

Comment:
```suggestion
STRICT_ENFORCEMENT_DEFAULT_ROLES = ["admin", "operator", "read-only"]
```
to reduce confusion for consumers that haven't configured strict enforcement